runtime/lua/vim/treesitter/dev.lua

```diff
@@ -595,9 +600,10 @@ function M.edit_query(lang)
   end
   vim.cmd(cmd)
 
-  local parser = vim.treesitter._get_parser(buf, lang)
+  local parser = vim.treesitter.get_parser(buf, lang, { error = false })
```

Comment:
could `assert` here a well

Reply:
I thought `edit_query` was meant to return `nil, err_msg` while `query.edit` should do the actual assertion?

Reply:
Right, sorry, I keep getting these two mixed up. I see now that you return here rather than `error`, so this is fine of course.